code/style.css

```diff
@@ -1,4 +1,341 @@
 html {
-  background: #0025ff;
+  font-family: 'Raleway', sans-serif;
+}
+
+body {
+  margin: 0;
+  background-image: url("./pattern.png");
+}
+
+h1 {
+  color: #fff;
+  font-size: 30px; 
+  letter-spacing: 4px;
+}
+
+h2 {     
+  letter-spacing: 2px;
+  font-size: 30px;
+  margin-left: 16px;
+}
+
+h3 {
+  font-size: 16px;
+  letter-spacing: 2px;
+}
+
+
+/*** HEADER ***/
+
+.header {   
+  background: #000;
+  height: 100px;
+  display: flex;
+  align-items: center;
+  padding: 0 16px;
+  margin-bottom: 30px;
+}
+
+.logo {
+  width: 30px;
+  height: 30px;
+  margin-right: 20px;
+}
+
+.title {
+  text-decoration: none;
+} 
+
+.navigation {
+  display: none;
+}
+
+.navigation.open-menu {
+  position: absolute;
+  width: 100%;
+  height: 30%;
+  top: 70px;
+  left: 0;
+  background: #000;
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: space-evenly;
+  align-items: center;
+  animation-name: menu-animation;
+  animation-duration: 0.5s;
+}
+
+@keyframes menu-animation {
+  0% {
+    left: 100%;
+  }
+  100% {
+    left: 0;
+  }
+}
+
+.navigation a {
   color: #fff;
+  text-decoration: none;
+  margin-left: 16px;
+}
+
+.navigation a:hover, .navigation a:focus {
+  color: rgb(189, 189, 189);
+}
+
+.ham {
+  margin-left: auto;
+  background-color: transparent;
+  border: none;
+  margin-right: 10px;
+}
+
+.hamburger {        
+  width: 30px;
+  height: 25px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  position: relative;
+}
+
+.hamburger span {
+  width: 30px;
+  height: 4px;
+  background: #fff;
+  border-radius: 25%;
+}
+
+.close-menu span:nth-child(1) {
+  transform: rotate(135deg);
+  position: absolute;
+  top: 10px;
+  left: 0px;
+}
+
+.close-menu span:nth-child(2) {
+  display: none;
+}
+
+.close-menu span:nth-child(3) {
+  transform: rotate(-135deg);
+  position: absolute;
+  top: 10px;
+  left: 0px;
+}
+
+.social-media {
+  display: none;
+}
+
+.social-media a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.social-media a:hover, .social-media a:focus {
+  color: rgb(189, 189, 189);
+}
+
+
+/*** MAIN ***/
+
+main {
+  width: 90%;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+
+/*** ZOOM ON HOVER ***/
+
+.container:hover {
+  transform: scale(1.1);
+}
+
+
+/*** RESPONSIVE IMAGE ***/
+
+.responsive-image {
+  width: 100%;
+  height: auto;
+}
+
+/*** LINKS ***/
+
+.container a {
+  text-decoration: none;
+  color: #000;
+}
+
+
+/*** BIG NEWS ***/
+
+.big-news {
+  padding: 0px;
+}
+
+.big-news .container {
+  padding: 30px 16px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid rgb(228, 225, 225);
+  margin: 20px 0px;
+  background: #fff; 
+  box-shadow: 5px 5px #e7e6e6;
+}
+
+.big-news-text p {
+  font-weight: bold;
+  letter-spacing: 2px;
+  font-size: 20px;
+  margin-left: 16px;
+}
+
+
+/*** OTHER NEWS ***/
+
+.other-news {    
+  display: flex;
+  flex-direction: column;
+  width: 90%;
```

Comment:
Would be nice if this section was as wide as the big news section for it to be aligned :)